testing/todo.test.js

```diff
@@ -0,0 +1,53 @@
+const todoList = (title, description, dueDate, priority) => ({
+  title,
+  description,
+  dueDate,
+  priority,
+});
```

Comment:
- [ ] [REQUIRED] Please import this function instead of defining it in this file